python-lib/ontology_tagger.py

```diff
@@ -9,7 +10,15 @@
 from time import perf_counter
 import logging
 from sentence_splitter import SentenceSplitter
+from language_support import (
+    SPACY_LANGUAGE_LOOKUP,
+    SPACY_LANGUAGE_RULES,
+    SPACY_LANGUAGE_MODELS_LEMMATIZATION,
+)
 
+#Display spacy logs messages only for ERROR messages, not for WARNING
+logger = logging.getLogger("spacy")
+logger.setLevel(logging.ERROR)
```

Comment:
Need to investigate when warnings may appear before changing spacy default logging

Reply:
The warning appearing in the log is : 
`[Warning W108]  "The rule-based lemmatizer did not find POS annotation for the token #'token' .Check that your pipeline includes components that assign token.pos, typically 'tagger'+'attribute_ruler' or 'morphologizer' "`

Those warnings appear when the lemmatization is activate with languages "bn", "el", "fa" that are languages using lemma_rules from spacy-lookups-data, in monolingual or multilingual case. It causes one warning per keyword tokenized, that appears twice in dss logs as follow:

```
[21:42:56] [INFO] [dku.utils]  - [2021-05-05 21:42:56,138] [WARNING] [W108] The rule-based lemmatizer did not find POS annotation for the token 'sin'. Check that your pipeline includes components that assign token.pos, typically 'tagger'+'attribute_ruler' or 'morphologizer'.
[21:42:56] [INFO] [dku.utils]  - 2021-05-05 21:42:56,138 WARNING [W108] The rule-based lemmatizer did not find POS annotation for the token 'sin'. Check that your pipeline includes components that assign token.pos, typically 'tagger'+'attribute_ruler' or 'morphologizer'.
```

WDYT? Should I let the warnings ?

Reply:
Thanks for investigating.  I don't expect Greek, Persian, or Bengali to be the majority cases among our user base. To me, it's a bit extreme to deactivate all spacy warnings for this plugin just for 3 languages.

Can you deactivate these warnings only if these languages are present in the dataset, or selected in the UI?

If not, I would vote to keep the logging of spaCy as it is, so display the warnings.

Reply:
I have deactivated the warnings for the specific case that bother us (e.g when lemmatization is activated and we have at least one of the 3 above languages) 2997cb7